docs/styleguide.md

```diff
@@ -39,4 +39,4 @@ When referring to a project on GitHub, link to the repository so others can dive
 
 ## More guidance
 
-Understand our [content model](content-model.md) and [audience](personas.md)
\ No newline at end of file
+Understand our [content model](content-model.md) and [audience](personas.md)
```

Comment:
What changed here?

Reply:
I didn't made any change here, I think there may be a shift in space

Reply:
Ok, np!